Client/Ecosia/Analytics/Analytics.Values.swift

```diff
@@ -4,125 +4,173 @@ extension Analytics {
     enum Category: String {
         case
         activity,
-        abTest = "ab_Test",
```

Comment:
Isn't this needed anymore?

Reply:
It's not being used anywhere in code, so no reason to keep it imo. If we ever need it again we can re-add.